cogs/customreactions.py

```diff
@@ -0,0 +1,1111 @@
+                    title = "Current custom reaction proposals\n" \
+                            "{}: Write the number of the custom reaction proposal you want to " \
+                            "approve, reject, edit, or see more information on".format(user_modifying)
+                else:
+                    title = "Current custom reactions\n" \
+                            "{}: Write the number of the custom reaction you want to edit or see more " \
+                            "information on".format(user_modifying)
+                message.embeds[0].title = title
+                await message.edit(embed=message.embeds[0])
+                number = 0
+                try:
+                    number_sent_by_user_modifying = False
+                    manual_timeout = time.time() + 60
+                    while not number_sent_by_user_modifying:
+                        msg = await self.bot.wait_for('message',
+                                                      check=number_check,
+                                                      timeout=60)
+                        if msg.author == user_modifying:
+                            number = int(msg.content)
+                            number_sent_by_user_modifying = True
+                            await msg.delete()
+                        else:
+                            if time.time() > manual_timeout:
+                                raise asyncio.TimeoutError
+
+                except asyncio.TimeoutError:
+                    pass
+
+                if number < 1 or number > len(current_list):
+                    if proposals:
+                        title = "Current custom reaction proposals\nClick on 🆗 to approve, reject, edit, or see " \
+                                "more information on one of them (Previous attempt received invalid input or timed out)"
+                    else:
+                        title = "Current custom reactions\nClick on 🆗 to edit or see more information on one of " \
+                                "them (Previous attempt received invalid input or timed out)"
+                    p = Pages(ctx,
+                              msg=message,
+                              item_list=reaction_dict,
+                              title=title,
+                              display_option=(2, 10),
+                              editable_content_emoji="🆗",
+                              return_user_on_edit=True)
+                else:
+                    await information_on_react(message, current_list, number,
+                                               proposals)
+                    if proposals:
+                        title = "Current custom reaction proposals\nClick on 🆗 to approve, reject, edit, or see " \
+                                "more information on one of them"
+                    else:
+                        title = "Current custom reactions\nClick on 🆗 to edit or see more information on one of them"
```

Comment:
duplicated string literals – use variables

Reply:
- "Current custom reaction proposals\nClick on 🆗 to approve, reject, edit, or see more information on one of them"

- "Current custom reaction proposals\nClick on 🆗 to edit or see more information on one of them"

- "(Previous attempt received invalid input or timed out)"

Reply:
See https://github.com/idoneam/Canary/pull/273#issuecomment-656869764